settings.gradle.kts

```diff
@@ -1,19 +1,31 @@
-rootProject.name = "sample-materials-shop"
-
-include(":app")
-
 pluginManagement {
     repositories {
-        maven(url = "https://dl.bintray.com/kotlin/kotlin-eap/")
+        google {
+            content {
+                includeGroupByRegex("com\\.android.*")
+                includeGroupByRegex("com\\.google.*")
+                includeGroupByRegex("androidx.*")
+            }
+        }
+        mavenCentral()
         gradlePluginPortal()
-        google()
     }
+}
 
-    resolutionStrategy {
-        eachPlugin {
-            if (requested.id.id == "com.android.application") {
-                useModule("com.android.tools.build:gradle:${requested.version}")
+dependencyResolutionManagement {
+    repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
+    repositories {
+        google {
+            content {
```

Comment:
Why are you adding those filters btw?

Reply:
This will speed up dependency downloading, and it's the default template generated from the newer Android Studio.